components/popover/popover.jsx

```diff
@@ -207,6 +210,10 @@ class Popover extends React.Component {
 			PropTypes.object,
 			PropTypes.string,
 		]),
+		/**
+		 * Determines the type of the popover. `error` and `warning` allows the  content body to scroll. _Tested with snaphots._
```

Comment:
Not sure if always implied, but we could add `Defaults to "base"` in the comment.

Reply:
I'll add it. It's also in the default props which should show up on the doc site